assets/src/apps/delivery/layouts/deck/RestartLessonDialog.tsx

```diff
@@ -62,7 +62,7 @@ const RestartLessonDialog: React.FC<RestartLessonDialogProps> = ({ onRestart })
       const newAttemptUrl = `/sections/${sectionSlug}/page/${revisionSlug}/attempt`;
       redirectTo = newAttemptUrl;
     }
-    if (!graded || isPreviewMode) {
+    if (isPreviewMode) {
```

Comment:
Hey @rmathis, please review.

If the lesson is not graded then just reloading the windows does not restart the lesson. the correct url logic is already in place (i.e. `newAttemptUrl` ).